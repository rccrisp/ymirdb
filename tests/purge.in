set a 1
snapshot
set a 2
snapshot
set a 3
snapshot
checkout 1
get a
purge a
get a
list snapshots
checkout 2
get a
checkout 3
get a
purge a
purge b
bye