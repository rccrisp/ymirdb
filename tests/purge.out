> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> [1]

> ok

> no such key

> 3
2
1

> ok

> no such key

> ok

> no such key

> ok

> ok

> bye
